package.json

```diff
@@ -111,68 +111,68 @@
     "npm-run-all": "^4.1.5",
     "postcss-loader": "^3.0.0",
     "rimraf": "^2.6.2",
-    "sass-loader": "^7.1.0",
+    "sass-loader": "^7.3.1",
```

Comment:
This may be a good time to consider switching from `node-sass` to `sass` as well, which is usually a drop-in replacement and doesn't have the compatibility issues that `node-sass` presents due to its binary implementation (`sass` is JavaScript).